portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +25,33 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private ArrayList<String> messages;
```

Comment:
List<String> messages should work fine.

Reply:
Responded in the comment below: For my implementation of doPost in a later pull request, I appended to the ArrayList messages. In this case, would it be better to switch to ImmutableList and create a new list every time in doPost or continue using ArrayList?